tests/test_amity_success.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from app.amity import Amity
+from collections import defaultdict
+
+
+class ClassAmitySuccessTest(unittest.TestCase):
+    def setUp(self):
+        self.amity = Amity()
+
+    def test_add_person(self):
+        self.amity.all_people = []
+        self.assertEqual(len(self.amity.all_people), 0)
+        self.amity.add_person("awesome", "fellow","y" )
+        self.assertNotEqual(len(self.amity.all_people), 0)
+        self.amity.all_people = []
```

Comment:
Its advisable to use a teardown method to create isolated test rather that resetting the same values over and over.